private/migrate/versions_test.go

```diff
@@ -89,9 +89,30 @@ func basicMigration(t *testing.T, db *sql.DB, testDB migrate.DB) {
 		},
 	}
 
+	dbVersion, err := m.CurrentVersion(nil, testDB)
+	assert.NoError(t, err)
+	assert.Equal(t, dbVersion, -1)
```

Comment:
why does ensureVersionTable fail error in this case?

Reply:
oh I see, the migration has yet to run

Reply:
yeah. it doesn't error, `getLatestVersion` returns a -1 which means that the migration hasn't run yet